### ISSUE TITLE
Could you provide a ngnix config demo

### ISSUE THREAD

**Issue by JayMiao** [mlazarov/supervisord-monitor#23]
Excuse my newbie. I've use this monitor under the ngnix config.
`server {
    listen 80;
    server_name supervisord.add.net;
    root /home/q/server/supervisord-monitor/public_html;
    index  index.php;

    location = /ENV {
        allow 127.0.0.1;
        deny all;
    }

    location ~* \.php$ {
        include fastcgi.conf;
        include include/supervisord.add.net/env.conf;
        fastcgi_pass 127.0.0.1:9000;
        fastcgi_index index.php;
    }
}`

But, when I choose to stop/start a progress. The got a ngnix error code : 404
Could you please provide a ngnix config file demo for you project ? THANKS

**Comment by mlazarov** [comment]
To work supervisord-monitor requires one additional `location` directive. This is how your vhost will look after adding it:
```
location = /ENV {
    allow 127.0.0.1;
    deny all;
}

location / {
    try_files $uri $uri/ /index.php;
 }

location ~* \.php$ {
    include fastcgi.conf;
    include include/supervisord.add.net/env.conf;
    fastcgi_pass 127.0.0.1:9000;
    fastcgi_index index.php;
}
```

**Comment by JayMiao** [comment]
Yeah ! I just resolve the 404 problem according to Ngnix Official Doc. 
I find the project written in CI framework.
Could you please put the link [CI ngnix](https://www.nginx.com/resources/wiki/start/topics/recipes/codeigniter/) in your README.md ?
